### PR TITLE
This PR will allow FindQGIS.cmake to find QGIS libs/includes in QGIS_PREFIX_PATH too

### PR DESCRIPTION
**Pull request by timlinux**

### CODE REVIEW

**Comment by m-kuhn** [comment]
As discussed, it may be better to implement this as a CMake option instead of a environment variable to make the difference runtime/buildtime clear.

**Comment by timlinux** [comment]
Thanks @m-kuhn I updated the PR accordingly

File: cmake/FindQGIS.cmake [hunk 1]
```diff
@@ -94,6 +95,7 @@ ELSE(WIN32)
     FIND_PATH(QGIS_INCLUDE_DIR
       NAMES qgis.h
       PATHS
+        {QGIS_PREFIX_PATH}/include/qgis
```
Comment (m-kuhn) [inline cmake/FindQGIS.cmake]: There should also be a `$`, no?

**Comment by m-kuhn** [comment]
Thanks @timlinux 
Not sure how CMake handles undefined variables. Is it ok to leave ${QGIS_PREFIX_PATH} undefined?

**Pull request merged**